symplyphysics/laws/dynamics/torque_due_to_twisting_force.py

```diff
@@ -0,0 +1,43 @@
+from sympy import Eq, solve, sin
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+
+# Description
+## Torque is a turning action on a body about a rotation axis due to a force.
+
+# Law: tau = F * r * sin(phi)
+## tau - torque
+## F - force exerted at the given point
+## r - distance to axis from the given point
+## phi - angle between the position vector and force vector
```

Comment:
What is `position vector`?

Reply:
In Russian terminology it's mostly known as radius vector, which is how it is sometimes referred to in English. In most English sources I found it to be called [position vector](https://www.sciencedirect.com/topics/mathematics/position-vector)

Reply:
Or do you mean the position vector of what thing I am referring to here?

Reply:
Yes, add short explanation, how to find position vector.